atlp-rwanda/script-heroes-frontend
# #175718972 Fix Landing Page Responsiveness

#### What does this PR do?
- Make a header responsive to all devices
- Make a footer responsive to all devices
- Make the main page responsive to all devices
#### Description of Task to be completed?
- Make a header responsive to all devices
- Make a footer responsive to all devices
- Make the main page responsive to all devices
-Revise Tests
#### How should this be manually tested?
- Clone the repo `git clone https://github.com/atlp-rwanda/script-heroes-frontend.git`
- Switch your working directory `cd script-heroes-frontend`
- checkout the branch `git checkout ft-landing-page-#175718972 `
- Install all dependencies `yarn install`
- Start the server `yarn run dev`
- Check responsiveness to different screen sizes
#### What are the relevant pivotal tracker stories?
[#175718972](https://www.pivotaltracker.com/n/projects/2465784)
